.github/workflows/security-scan.yml

```
@@ -6,7 +6,9 @@ on:
   pull_request:
     branches:
       - 'main'
-
+    paths-ignore:
+      - 'website/'
```

Not sure if this needs the `**` here: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-excluding-paths
```suggestion
      - 'website/**'
```

---

Ah, yes. Let me get a new PR up since I was quick on the trigger here.

---

@moduli https://github.com/hashicorp/boundary/pull/3435